### ISSUE TITLE
Cannot unset default InverseMethods in Style/InverseMethods

### ISSUE THREAD

**Issue by dduugg** [rubocop-hq/rubocop#6926]
The default setting of `Style/InverseMethods` includes:
```yaml
Style/InverseMethods:
  InverseMethods:
    :any?: :none?
    :even?: :odd?
    :==: :!=
    :=~: :!~
    :<: :>=
    :>: :<=
```
I would like to unset the `:<` and `:>` entries, as the inverse methods are not correct when comparing classes, e.g.
```ruby
!(Integer < String)
```
is not equivalent to
```ruby
Integer >= String
```
In reading https://github.com/rubocop-hq/rubocop/issues/4289 as well as https://github.com/rubocop-hq/rubocop/blob/master/manual/configuration.md#inheritance-of-hashes-vs-other-types my understanding is that i should be able to unset them with:
```yaml
Style/InverseMethods:
  InverseMethods:
    :<: ~
    :>: ~
```
--------

## Expected behavior

I would expect the above to result in (via `rubocop --show-cops`):

```yaml
Style/InverseMethods:
  InverseMethods:
    :any?: :none?
    :even?: :odd?
    :==: :!=
    :=~: :!~
```

## Actual behavior

Instead the result is:
```yaml
Style/InverseMethods:
  InverseMethods:
    :any?: :none?
    :even?: :odd?
    :==: :!=
    :=~: :!~
    :<:
    :>:
```

This will continue to erroneously flag code such as `!(a < b)` (which, in addition, is autocorrected to `a  b`).

## Steps to reproduce the problem
 
I have written a failing [test](https://github.com/dduugg/rubocop/commit/e43f7b87eb3ae7dbe8745eecaaff2045f72adc12) to capture the desired behavior: 
```ruby
context 'when a file inherits and overrides a hash with nil' do
  let(:file_path) { '.rubocop.yml' }

  before do
    create_file('.rubocop_parent.yml', <<-YAML.strip_indent)
      Style/InverseMethods:
        InverseMethods:
          :any?: :none?
          :even?: :odd?
          :==: :!=
          :=~: :!~
          :<: :>=
          :>: :<=
    YAML

    create_file('.rubocop.yml', <<-YAML.strip_indent)
      inherit_from: .rubocop_parent.yml

      Style/InverseMethods:
        InverseMethods:
          :<: ~
          :>: ~
          :foo: :bar
    YAML
  end

  it 'removes hash keys with nil values' do
    inverse_methods =
      configuration_from_file['Style/InverseMethods']['InverseMethods']
    expect(inverse_methods).to eq(
      '==': :!=,
      '=~': :!~,
      any?: :none?,
      even?: :odd?,
      foo: :bar
    )
  end
end
```

## RuboCop version
0.67.2
